client.js

```diff
@@ -45,100 +42,338 @@ bot.on('message', function (message) {
+                                    }
+                                    const collector = message.createReactionCollector(react_filter, { time: 1000 * 60 * 60 * 24 , dispose: true});
+                                    collector.on('collect', (reaction, user) => {
+                                        switch (reaction.emoji.name) {
+                                            case '💤':
+                                                // Snooze this watch for 6 hours
+                                                db.snooze('watch', watch.id);
+                                                user.send(`Sleep is good.  Pausing notifications for the next 6 hours on your \`\`${item}\`\` watch on \`\`${server}\`\`.  Click 💤 again to unsnooze.  To snooze all watches, use \`\`!snooze\`\``).catch(console.error);
+                                                break;
+                                            case '❌':
+                                                // Delete this watch
+                                                db.endWatch(null, null, null, watch.id);
+                                                user.send(`Very well, no longer watching for auctions of \`\`${item}\`\`\ on \`\`${server}\`\`.`);
+                                                break;
+                                            case '♻': //extend watch
+                                                db.extendWatch(watch.id)
+                                                user.send(`Good things come to those who wait.  I added another 7 days to your \`\`${item}\`\` watch on \`\`${server}\`\`.`);
+                                                break;
+                                            default:
+                                                break;
+                                        }
+                                    })
+                                    collector.on('remove', (reaction, user) => {
+                                        switch (reaction.emoji.name) {
+                                            case '💤':
+                                                // unsnooze watch
+                                                db.unsnooze('watch', watch.id);
+                                                user.send(`Rise and grind.  No longer snoozing on your \`\`${item}\`\` watch on \`\`${server}\`\`.`).catch(console.error);
+                                                break;
+                                            case '❌':
+                                                // Renew watch
+                                                db.addWatch(null, null, null, null, watch.id);
+                                                user.send(`Ok, watching for auctions of \`\`${item}\`\` on P1999 \`\`${server}\`\` again.`);
+                                                break;
+                                            case '🔕':
```

Comment:
This feature is cool, but isn't this the wrong place for this? It looks like this took the place of "un-refresh", which is fine because that seems like a kinda weird/nonsensical option, but this should really be on an individual watch notification, right? Not the watch show/list command?

Reply:
Good catch, copy/pasta gone wrong.  Removing the "remove" event listener for  '🔕' which doesn't belong here anyway..